org.getmonero.Monero.yaml

```diff
@@ -18,10 +18,11 @@ modules:
       url: https://github.com/monero-project/monero-gui/archive/refs/tags/v0.17.1.9.tar.gz
       sha256: 3a924b4e08b557d93337a0c05826a9fa95e5cf8c503aff0ecb3676be45db1d14
     build-commands:
-    - tar xzf v0.17.1.9.tar.gz
-    - cd monero-gui-0.17.1.9
-    - docker build --tag monero:build-env-linux --build-arg THREADS=4 --file Dockerfile.linux .
-    - docker run --rm -v $PWD:/monero-gui -w /monero-gui monero:build-env-linux sh -c 'make release-static -j4'
+    - sudo apt install build-essential cmake libboost-all-dev miniupnpc libunbound-dev graphviz doxygen libunwind8-dev pkg-config libssl-dev libzmq3-dev libsodium-dev libhidapi-dev libnorm-dev libusb-1.0-0-dev libpgm-dev libprotobuf-dev protobuf-compiler libgcrypt20-dev
+    - sudo apt install qtbase5-dev qt5-default qtdeclarative5-dev qml-module-qtqml-models2 qml-module-qtquick-controls qml-module-qtquick-controls2 qml-module-qtquick-dialogs qml-module-qtquick-xmllistmodel qml-module-qt-labs-settings qml-module-qt-labs-folderlistmodel qttools5-dev-tools qml-module-qtquick-templates2 libqt5svg5-dev
+    - git clone --recursive https://github.com/monero-project/monero-gui.git
+    - cd monero-gui
+    - make release -j $FLATPAK_BUILDER_N_JOBS
```

Comment:
This is not how it is done. It won't work either. There is no apt-get or debian package in the platform.

Dependencies are listed as modules in the manifest, one by one.

Reply:
Also since this require Qt, you should build against the KDE platform.